oidc-api/src/test/java/uk/gov/di/authentication/oidc/services/InitiateIPVAuthorisationServiceTest.java

```diff
@@ -312,7 +360,7 @@ public static Map<String, String> splitQuery(String stringUrl) {
         return query_pairs;
     }
 
-    private AuthenticationRequest createAuthenticationRequest() {
+    private AuthenticationRequest createAuthenticationRequest(ClaimsSetRequest claimsSetRequest) {
```

Comment:
I might be misreading this, but I think it would be nice to change this variable name, as it seems to be the same as the thing we're passing in?

Reply:
Have updated!